bin/alluxio-start.sh

```diff
@@ -85,25 +83,26 @@ check_mount_mode() {
     Mount);;
     SudoMount);;
     NoMount)
-      is_ram_folder_mounted
+      local folder_type=$(${BIN}/alluxio getConf alluxio.worker.tieredstore.level0.alias)
+      local ram_folder=$(${BIN}/alluxio getConf alluxio.worker.tieredstore.level0.dirs.path)
+      if [[ ${folder_type} != "MEM" ]]; then
```

Comment:
no guarantee. if not called "MEM", no check enforced --- not known how to interpret the type any way.